incubator/gocd/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+name: gocd
+home: https://gocd.org/
+version: 0.1.0
+appVersion: 17.12.0
+description: A GoCD Helm chart for Kubernetes. GoCD is an Open source continuous delivery server to model and visualize complex workflows with ease.
+icon: https://gocd.github.io/assets/images/go-icon-black-192x192.svg
+keywords:
+- gocd
+- ci
+- cd
+- ci/cd
+- continuous-integration
+- continuous-delivery
+- continuous-deployment
+- continuous-testing
+sources:
+- https://github.com/gocd/gocd
+maintainers:
+- name: ThoughtWorks, Inc.
```

Comment:
Use Github username.

Reply:
@unguiculus -- GoCD Helm Chart is done by [`GoCD`](https://github.com/gocd/gocd) development team. Everyone from the team will be maintaining this Helm chart.

Is there any specific reason for having Github username as the maintainer name?

Reply:
@GaneshSPatil There are a few reasons for using GitHub usernames.

1. It lets us identify who the chart owners are as people. We can see who comments and has an opinion on GitHub and if they are a chart maintainer or not. This is also useful when we cc people in issues to know the GitHub names to use.
2. We recently instituted OWNERS files on this repo. This has the ability for people to be notified for review of a PR automatically and to have the ability to merge PRs for this chart.  [I recently wrote about it](https://codeengineered.com/blog/2017/kubernetes-chart-owners/). Eventually we'd like to have folks start using an OWNERS file for this chart. The names in them are the GitHub usernames and to have that tracibility would be useful.

I fully respect that this is a company project by ThoughtWorks. It's just that we need a way to interact with people who are involved with maintaining the chart.

Reply:
@mattfarina -- Thanks for the explanation, I will change the maintainers' names accordingly.